src/OlStyleParser.spec.ts

```diff
@@ -150,6 +159,24 @@ describe('OlStyleParser implements StyleParser', () => {
           expect(olImage.getFill().getColor()).toEqual(expecSymb.color);
         });
     });
+    it('can write a OpenLayers IconSymbolizer', () => {
+      expect.assertions(6);
+      return styleParser.writeStyle(point_icon)
+        .then((olStyles: ol.style.Style[]) => {
+          expect(olStyles).toBeDefined();
+
+          const expecSymb = point_icon.rules[0].symbolizer as IconSymbolizer;
+          const olImage: ol.style.Icon = olStyles[0].getImage() as ol.style.Icon;
```

Comment:
IMHO this is a doubled type declaration for `ol.style.Icon`

Reply:
I think the variable name is misleading. I'll rename it to `olIcon`.
The `… as ol.style.Icon` part is just a cast.